tests/model/api/test_http_api_generator.py

```diff
@@ -250,15 +250,18 @@ def test_basepaths(self):
         self.kwargs["domain"] = {
             "DomainName": "example.com",
             "CertificateArn": "some-url",
-            "BasePath": ["one-1", "two_2", "three", "/"],
+            "BasePath": ["one-1", "two_2", "three", "/", "api/v1"],
```

Comment:
We should add: `"/api"`, `"api/v1/"`, `"/api/v1/"` cases to test the `/` stripping.